### DISCUSSION TITLE
older versions of Headless documentation

### DISCUSSION THREAD

**Discussion by newtmitch** [tailwindlabs/headlessui#1738] [Help]
Hi - I have a version of Headless UI that's slightly older than the docs currently available online and it appears that the API for some of the components have changed. The doc site doesn't have a version switcher or anything like that and I can't seem to find the site repo or documentation itself in one of the available tailwindlabs repos. Is there any way to get to an older version of the docs for reference?

**Comment by kiettrantuan** [comment]
Hi @newtmitch , Did you find out how to read older version docs or upgrade guide ? If yes please share, thanks.
Currently I'm using web archive to read older version docs.
